src/components/CookieConsent.tsx

```diff
@@ -0,0 +1,73 @@
+import { posthog } from "posthog-js";
+import { useEffect, useState } from "react";
+
+const CookieConsent = () => {
+  const [show, setShow] = useState<boolean>(false);
+  const [optionSelected, setOptionSelected] = useState<boolean>(false);
+
+  useEffect(() => {
+    posthog.init("phc_OPJtdGL4gAGdo8VKLsHz4LmKfoOMKkrza1BsBNeUdx4", {
```

Comment:
```suggestion
    posthog.init("phc_qZk1aHPpXUq0ohRcTjFL5w8Tpo5Asht9UhFBjzruBFD", {
```

Might as well update the project while we're here.

Reply:
Update Posthog project key